src/sidebars/version-directive.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.version')
+        .value('version', '0.1.0')
```

Comment:
Added the bumper. I've kept it separated from `deploy`, since I can't verify it works, but `grunt release` bumps the version.